msal-auth-node/README.md

```diff
@@ -0,0 +1,129 @@
+| **Name**                          | `Node Web API`                                                               | Suggested value for this sample. <br/> You can change the app name at any time.                    |
+| **Supported account types**    | **Accounts in this organizational directory only (Single tenant)**   | Suggested value for this sample.                                                 |
+| **Platform type**              | _None_                                                               | No redirect URI required; don't select a platform.                               |
+| **Scopes defined by this API** | **Scope name**: `Greeting.Read`<br/>**Who can consent?**: **Admins and users**<br/>**Admin consent display name**: `Read API Greetings`<br/>**Admin consent description**: `Allows the user to see greetings from the API.`<br/>**User consent display name**: `Read API Greetings`<br/>**User consent description**: `Allows you to see greetings from the API.`<br/>**State**: **Enabled** | Required scope for this sample. |
+
+
+> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+
+
+
+### 4. Update code sample with app registration values
+
+```javascript
+app.use(jwt({
+  secret: jwks.expressJwtSecret({
+
+    // JWKS URI in the form of: https://login.microsoftonline.com/<tenant>/discovery/v2.0/keys
+    jwksUri: ''
+  }),
+  
+  // 'Application (client) ID' of app registration in Azure portal - this value is a GUID
+  audience: '',
+  
+  // Full directory URL, in the form of: https://login.microsoftonline.com/<tenant>/v2.0
+  issuer: '',
+
+  algorithms: ['RS256']
+}))
+```
+
+
+
+### 3. Install package(s)
+
+To install Node.Js libraries into your (virtual) environment:
+
+```bash
+npm install
+```
+
+## Run the application
+
+```bash
+node app.js
+```
+
+## Access the API
+
+Open postman, curl, or similar and make an HTTP GET request to **http://localhost:8080** with an `Authorization` header of `Bearer {valid-access-token}`. If everything worked, the sample app should produce output similar to this:
+
+```console
+$ curl http://localhost:8080 -H "Authorization: Bearer {valid-access-token}"
+Hello, world. You were able to access this because you provided a valid access token with the Greeting.Read scope as a claim.
+```
+
+
+
+## About the code
+
+This Node.Js API has a single route that requires an access token. The access token will be automatically validated by MSAL. A missing or invalid (expired, wrong audience, etc) token will result in a `401` response. An otherwise valid token without the proper scopes will result in a `403` response. A valid token with a proper scope of (`Greeting.Read`) will result in a "Hello, world." message.
+
```

Comment:
```suggestion
This Node.Js API has a single route that requires an access token. The access token will be automatically validated by MSAL. A missing or invalid (expired, wrong audience, etc) token will result in a `401` response. An otherwise valid token without the proper scopes will result in a `403` response. A valid token with a proper scope of (`Greeting.Read`) will result in a "Hello, world" message.
```